runbotics/runbotics-orchestrator-ui/src/components/InfoPanel/ProcessInstanceEventsDetails/ProcessInstanceEventsDetailsHeader.tsx

```diff
@@ -9,31 +9,23 @@ import AccessTimeIcon from '@mui/icons-material/AccessTime';
 import {
     GridContainer, GridItem,
 } from './ProcessInstanceEventsDetails.styles';
-import {checkIfKeyExists, translate} from '../../../hooks/useTranslations'
-import { convertToPascalCase } from 'src/utils/text';
+import useTranslations, { checkIfKeyExists } from '../../../hooks/useTranslations'
+import { capitalizeFirstLetter} from 'src/utils/text';
 import i18n from 'i18next';
-import { TranslationsDescriptors } from 'src/translations/translations';
 
 interface Props {
     processInstanceEvent: IProcessInstanceEvent;
 }
 
 const ProcessInstanceEventsDetailsHeader: VFC<Props> = ({ processInstanceEvent }) => {
-    const formattedStatus = convertToPascalCase(processInstanceEvent.status);
-    const [step, setStep] = useState<string>(processInstanceEvent.step)
-
-    useEffect(() => {
-        if (checkIfKeyExists(processInstanceEvent.step)) {
-            setStep(translate(processInstanceEvent.step));
-        } else {
-            setStep(processInstanceEvent.step);
-        }
-    }, [i18n.language])
-
+    const { translate } = useTranslations();
+    const formattedStatus = capitalizeFirstLetter({ text: processInstanceEvent.status, lowerCaseRest: true, delimiter: /_| / });
+    const translateKey = `Process.Details.Modeler.Actions.${capitalizeFirstLetter({ text: processInstanceEvent.step, lowerCaseRest: false, delimiter: '.', join: '.'})}.Label`;
```

Comment:
`lowerCaseRest` is false by default no need to include it, adjust all similar appearances

Reply:
done